webapp-django/crashstats/monitoring/views.py

```diff
@@ -81,9 +82,9 @@ def dockerflow_heartbeat(request):
     assert Permission.objects.all().count() > 0
 
     # We should also be able to set and get a cache value
-    cache_key = "__healthcheck__"
+    cache_key = "__healthcheck__" + str(uuid.uuid4())
```

Comment:
This creates a string like `__healthcheck__6a9b3d64-d68a-40d6-8801-8a6b4db27a9c`.